README.md

```diff
@@ -255,6 +259,110 @@ Here is an example:
+- `model_params.batch_size`: Batch size when running SPECTER. This defaults to 16.
+- `model_params.publications_path`: When running SPECTER, this is where the embedded abstracts/titles of the Reviewers (and Area Chairs) are stored.
+- `model_params.submissions_path`: When running SPECTER, this is where the embedded abstracts/titles of the Submissions are stored.
+- `model_params.average_score` (boolean, defaults to `false`): This parameter specifies that the reviewer is assigned based on the average similarity of the submission to the authored publication embeddings. Exactly one of `model_params.average_score` and `model_params.max_score` must be `true`.
+- `model_params.max_score` (boolean, defaults to `true`): This parameter specifies that the reviewer is assigned based on the max similarity of the submission to the authored publication embeddings. Exactly one of `model_params.average_score` and `model_params.max_score` must be `true`.
+- `model_params.skip_specter`: Since running SPECTER can take a significant amount of time, the vectors are saved in `model_params.submissions_path` and `model_params.publications_path`. The jsonl files will be loaded with all the vectors.
+
+Here is an example:
+```
+{
+    "name": "iclr2020_specter",
+    "dataset": {
+        "directory": "./"
+    },
+    "model": "specter",
+    "model_params": {
+        "specter_dir": "../specter/",
+        "work_dir": "./",
+        "average_score": false,
+        "max_score": true,
+        "use_cuda": true,
+        "batch_size": 16,
+        "publications_path": "./",
+        "submissions_path": "./",
+        "scores_path": "./"
+    }
+}
+```
+
+#### Multifacet-Recommender specific parameters (affinity scores):
+- `model_params.feature_vocab_file`: Path to the vocablary file for the text encoders
+- `model_params.model_checkpoint_dir`: Path to the directory containing the trained Multifacet-Recommender model checkpoints
+- `model_params.work_dir`: When running Multifacet-Recommender, this is where the intermediate files are stored.
+- `model_params.epochs`: Number of epochs to finetune reviewer embeddings. This defaults to 100.
+- `model_params.batch_size`: Batch size when running Multifacet-Recommender. This defaults to 50.
+- `model_params.use_cuda`: Boolean to indicate whether to use GPU (`true`) or CPU (`false`) when running Multifacet-Recommender. Currently, only 1 GPU is supported, but there does not seem to be necessary to have more.
+
+Here is an example:
+```
+{
+    "name": "iclr2020_mfr",
+    "dataset": {
+        "directory": "./"
+    },
+    "model": "mfr",
+    "model_params": {
+        "feature_vocab_file": "../mfr_data/feature_vocab_file",
+        "model_checkpoint_dir": "../mfr_data/mfr_model_checkpoint/",
+        "work_dir": "./",
+        "epochs": 100,
+        "batch_size": 50,
+        "use_cuda": true,
+        "scores_path": "./"
+    }
+}
+```
+
+#### SPECTER+MFR ensemble specific parameters (affinity scores):
```

Comment:
I'm testing specter+mfr and I'm getting this error:

```
3316batches [09:59,  5.53batches/s]
MFR:
Featurizing publications...
Traceback (most recent call last):
  File "/home/mbok/miniconda3/envs/specter/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/mbok/miniconda3/envs/specter/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/mbok/openreview-expertise/expertise/run.py", line 145, in <module>
    mfr_publications_path=None, skip_specter=config['model_params'].get('skip_specter', False))
  File "/home/mbok/openreview-expertise/expertise/models/multifacet_recommender/ensemble.py", line 58, in embed_publications
    self.mfr_predictor.embed_publications(mfr_publications_path)
  File "/home/mbok/openreview-expertise/expertise/models/multifacet_recommender/multifacet_recommender.py", line 610, in embed_publications
    abstracts_corpus_tokens = self._tokenize_text(abstracts_corpus)
  File "/home/mbok/openreview-expertise/expertise/models/multifacet_recommender/multifacet_recommender.py", line 297, in _tokenize_text
    w_list.append([w.text for w in nlp.tokenizer(line)])
  File "tokenizer.pyx", line 85, in spacy.tokenizer.Tokenizer.__call__
TypeError: object of type 'NoneType' has no len()

If you suspect this is an IPython 7.18.1 bug, please report it at:
    https://github.com/ipython/ipython/issues
or send an email to the mailing list at ipython-dev@python.org

You can print a more detailed traceback right now with "%tb", or use "%debug"
to interactively debug it.

Extra-detailed tracebacks for bug-reporting purposes can be enabled via:
    %config Application.verbose_crash=True

09/24/2020 10:04:11 - INFO - allennlp.models.archival -   removing temporary unarchived model dir at /tmp/tmpcdzpjyl2
```

Reply:
Pushed some changes. Can you try again?